config/toml.go

```diff
@@ -456,6 +456,12 @@ version = "{{ .FastSync.Version }}"
 #######################################################
 [consensus]
 
+# Experimental: if set to "true", only internal messages will be
+# written to the WAL. External messages like votes, proposal,
+# block parts, will not be written.
+# Use at your own risk!
```

Comment:
can we make an issue to remind us to delete the "Use at your own risk?" after we are confident that its working as intended? either that or we should remove it now and be very confident that its safe before we actually include it in any release candidate.

Reply:
Removed from the PR and made the default be `true`